### ISSUE TITLE
Java Mini Capstone [June 2020, ENSF593 ]

### ISSUE THREAD

**Issue by stancsz** [ensf/ensf.github.io#1]
- [ ] Brainstorm ideas

- [ ] Vote & decide on a project by June 24th, 2020

- [ ] Notify Yves & Mohammad to get some eyes-on/supervision

- [ ] Have a design meeting to breakdown the tasks into issues in the mini capstone repo

- [ ] Assign ourselves with interest issue(s) and implement work in branches

- [ ] Submit PR and ask for peer reviews when work is complete

Learning objectives:
Git & GitHub, Branching, PRs, Issues, Wiki

Looking forward with working with you all =)

**Comment by bgulseren** [comment]
What do you say for the stats calculator as the starter project?

**Comment by stancsz** [comment]
> What do you say for the stats calculator as the starter project?

Yep, looks like we'll be doing stats calculator. I think we should create a new repo for that.
